src/components/SideBar/index.js

```diff
@@ -138,9 +119,37 @@ class SideBar extends PureComponent<Props> {
   }
 }
 
+const AccountsList = connect(state => ({
+  accounts: getVisibleAccounts(state),
+}))(({ accounts }: { accounts: Account[] }) => (
+  <Fragment>
+    {accounts.map(account => {
+      const Icon = getCryptoCurrencyIcon(account.currency)
+      return (
+        <Item
+          big
+          desc={
+            <FormattedVal
+              alwaysShowSign={false}
+              color="graphite"
+              unit={account.unit}
+              showCode
+              val={account.balance || 0}
+            />
+          }
+          iconActiveColor={account.currency.color}
+          icon={Icon ? <Icon size={16} /> : null}
+          key={account.id}
+          linkTo={`/account/${account.id}`}
+        >
+          {account.name}
+        </Item>
+      )
+    })}
+  </Fragment>
+))
+
 export default compose(
-  connect(mapStateToProps, mapDispatchToProps, null, {
-    pure: false,
-  }),
+  connect(mapStateToProps, mapDispatchToProps, null, { pure: false }),
```

Comment:
why do we need to `{pure:false}` ?

Reply:
Because of the internal and mysterious behaviour of react-router combined with redux (& react-router-redux..)

see [here](https://github.com/reduxjs/react-redux/blob/master/docs/troubleshooting.md#my-views-arent-updating-when-something-changes-outside-of-redux) for more context